validate.py

```diff
@@ -33,11 +33,25 @@ def lower_headings(iterator):
     'error': 0,
     'warning': 0
   }
+  ids = set()
   for row in reader:
     num = reader.line_num
     errors = []
     warnings = []
 
+    ###
+    # Data Center ID should be present and unique.
+    ###
+
+    if not row.get('data center id'):
+      errors.append('Data Center ID must not be blank.')
+
+    elif row.get('data center id') in ids:
```

Comment:
IDs must only be unique for records that are valid.  Invalid records may end up with duplicate identifiers over time.